examples/NiO/se_e2_a/input.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
I suggest renaming the directory name from `NiO` to `spin` as it is more meaningful to others.